docs/docs/guides/tracking/feedback.md

```diff
@@ -3,33 +3,75 @@ import TabItem from '@theme/TabItem';
 
-## View and Add Feedback within UI
+This guide covers how to use Weave’s feedback functionality in both the UI and SDK, query and manage feedback, and use human annotations for detailed evaluations. 
 
-Reactions and notes are displayed in a column in the calls grid. Hovering over these indicators provides more detail. Use these buttons to add reactions or notes.
+- [Provide feedback in the UI](#provide-feedback-in-the-ui)
+- [Provide feedback via the SDK](#provide-feedback-via-the-sdk)
+- [Add human annotations](#add-human-annotations)
 
-![Screenshot of calls grid with feedback column](imgs/feedback_calls.png)
+## Provide feedback in the UI
 
-View and edit feedback in the header of the call details page.
+In the Weave UI, you can add and view feedback [from the call details page](#from-the-call-details-page) or [using the icons](#use-the-icons).
 
-![Screenshot of feedback controls in call details header](imgs/feedback_call_header.png)
+### From the call details page
 
-View the feedback for a call on the "Feedback" tab of the call details page.
+1. In the sidebar, navigate to **Traces**.
+2. Find the row for the call that you want to add feedback to.
+3. Open the call details page.
+4. Select the **Feedback** column for the call.
+5. Add, view, or delete feedback:
+   - [Add and view feedback using the icons](#use-the-icons) located  in the upper right corner of the call details feedback view.
+   - View and delete feedback from the call details feedback table. Feedback can be deleted by clicking the trash icon in the rightmost column of the appropriate feedback row.
 
 ![Screenshot of Feedback tab in call details](imgs/feedback_tab.png)
 
-Access copy-and-paste examples on the "Use" tab of the call details page to manipulate the feedback for that call using the SDK.
+### Use the icons
+
+You can add or remove a reaction, and add a note using the icons that are located in both the call table and individual call details pages.
+
+- *Call table*: Located in **Feedback** column in the appropriate row in the call table.
+- *Call details page*: Located in the upper right corner of each call details page.
+
+To add a reaction:
+  1. Click the emoji icon. 
+  2. Add a thumbs up, thumbs down, or click the **+** icon for more emojis.
+
+To add a comment:
+  1. Click the comment bubble icon. 
+  2. In the text box, add your note. 
+  3. To save the note, press the **Enter** key. You can add additional notes.
+
+To remove a reaction:
+  1. Hover over the emoji reaction you want to remove.
+  2. Click the reaction to remove it.
+
+![Screenshot of calls grid with feedback column](imgs/feedback_calls.png)
+
+## Provide feedback via the SDK
+
+> You can find SDK usage examples for feedback in the UI under the **Use** tab in the call details page.
 
-![Screenshot of Use tab in call details](imgs/feedback_use.png)
+You can use the Weave SDK to programmatically add, remove, and query feedback on calls. 
 
-## SDK
+### Query a project's feedback
```

Comment:
in general we have to be careful changing these headings, because they get included in any links in the app, which will then break when changed. Lets do it this time, i'll track down any links to these headers, but just an fyi.

Reply:
looks like only one place we have to change, i'll put up a pr to change to change it

Reply:
If it's just link headers, I can change track them down in this PR, no need for an extra